### PR TITLE
Remove unused code

### PR DESCRIPTION
**Pull request by kean**
This is one more sweep with Periphery, removing leftover code from the "Offline Mode" project and other unused code.

To test: n/a

## Regression Notes
1. Potential unintended areas of impact

2. What I did to test those areas of impact (or what existing automated tests I relied on)

3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:

- [ ] I have completed the Regression Notes.
- [ ] I have considered adding unit tests for my changes.
- [ ] I have considered adding accessibility improvements for my changes.
- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

Testing checklist:
- [ ] WordPress.com sites and self-hosted Jetpack sites.
- [ ] Portrait and landscape orientations.
- [ ] Light and dark modes.
- [ ] Fonts: Larger, smaller and bold text.
- [ ] High contrast.
- [ ] VoiceOver.
- [ ] Languages with large words or with letters/accents not frequently used in English.
- [ ] Right-to-left languages. (Even if translation isn’t complete, formatting should still respect the right-to-left layout)
- [ ] iPhone and iPad. 
- [ ] Multi-tasking: Split view and Slide over. (iPad)

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Pages/EditPageViewController.swift [hunk 1]
```diff
@@ -10,14 +10,14 @@ class EditPageViewController: UIViewController {
     var onClose: (() -> Void)?
 
     convenience init(page: Page) {
-        self.init(page: page, blog: page.blog, postTitle: nil, content: nil, appliedTemplate: nil)
+        self.init(page: page, blog: page.blog, postTitle: nil, content: nil)
```
Comment (kean) [inline WordPress/Classes/ViewRelated/Pages/EditPageViewController.swift:13]: This is one more step to consolidating `EditPostViewController` and `EditPageViewControleler`, which is great.

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">View files have been modified, but no screenshot or video is included in the pull request. Consider adding some for clarity.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WordPress/Classes/ViewRelated/Pages/PageListCell.swift [hunk 2]
```diff
@@ -113,6 +113,7 @@ final class PageListCell: UITableViewCell, AbstractPostListCell, PostSearchResul
         setupLabels()
         setupFeaturedImageView()
         setupEllipsisButton()
+        setupIcon()
```
Comment (kean) [inline WordPress/Classes/ViewRelated/Pages/PageListCell.swift:116]: This code was reported as unused, but it's a mistake. The issue was harmless: this icon was a tiny bit larger than expected:

File: WordPress/Classes/ViewRelated/Post/PostSettingsViewController+Swift.swift [hunk 3]
```diff
@@ -223,10 +223,7 @@ extension PostSettingsViewController {
             if let index = pages.firstIndex(of: page) {
                 pages = pages.remove(from: index)
             }
-            let viewController = ParentPageSettingsViewController.make(with: pages, selectedPage: page) { [weak self] in
-                self?.navigationController?.popViewController(animated: true)
-                self?.tableView.reloadData()
-            }
+            let viewController = ParentPageSettingsViewController.make(with: pages, selectedPage: page)
```
Comment (kean) [inline WordPress/Classes/ViewRelated/Post/PostSettingsViewController+Swift.swift:226]: The production behavior is that it shows you your selection, and then you need to tap "back" manually, like in most other details screen in settings.

**Review by crazytonyli — APPROVED** [review]

**Pull request merged**